hr_timesheet_sheet/README.rst

```diff
@@ -0,0 +1,80 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+HR Timesheet Sheet
+==================
+
+This module supplies a new screen enabling you to manage your work encoding (timesheet) by period.
+Timesheet entries are made by employees each day. At the end of the defined period,
+employees validate their sheet and the manager must then approve his team's entries.
+Periods are defined in the company forms and you can set them to run monthly, weekly or daily.
+
+Negative hours are not admitted.
+
+Installation
+============
+
+This module relies on:
+
+* The OCA module '2D matrix for x2many fields', and can be downloaded from
+  Github: https://github.com/OCA/web/tree/11.0/web_widget_x2many_2d_matrix
+
+
+Configuration
+=============
+
+If you want other default ranges different from weekly, you need to go:
+
+* In the menu `Configuration` -> `Settings` -> **Timesheet Options**,
+  and select in **Timesheet Sheet Range** the default range you want.
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/117/11.0
+
+Known issues / Roadmap
+======================
+
+* When you open the `Summary` or `Details` tab, a save should be performed
+to ensure the data shown is correct. This perhaps could be achieved by including
```

Comment:
Travis is failing because of this
```
************* Module hr_timesheet_sheet
hr_timesheet_sheet/README.rst:42: [E7901(rst-syntax-error), ]  Bullet list ends without a blank line; unexpected unindent.
```